redis/conn.go

```diff
@@ -142,6 +144,22 @@ func DialPassword(password string) DialOption {
 	}}
 }
 
+// DialUsername specifies the username to use when connecting to
+// the Redis server when Redis ACLs are used.
+func DialUsername(username string) DialOption {
+	return DialOption{func(do *dialOptions) {
+		do.username = username
+	}}
+}
+
+// DialUseACL specifies whether Redis ACL system should be used when connecting to the
+// server.
+func DialUseACL(useACL bool) DialOption {
+	return DialOption{func(do *dialOptions) {
+		do.useACL = useACL
```

Comment:
This doesn't seem to have any use apart from when combined with username, so is that sufficient i.e. if you the username is set its used?

Reply:
@stevenh I believe it might be necessary since we dont want to break compatibility wih previous url parsing ( that contained the user in the URL ). So I would stick to enforcing the user to aknowledge that he wants to use ACLs. WDYT?

Reply:
Given that would be trivial to fix, what I would say is a broken config anyway I think I'd prefer for simple use. In the case of a parsed url, you would need to enable this anyway as there is nothing else to tell otherwise.

Reply:
after asking for more advice youre right ( no  point on having this extra flag =) ). removing it

Reply:
solved in the latest commit :+1:

Reply:
> Given that would be trivial to fix, what I would say is a broken config anyway, I think I'd prefer to go for simple use.
> 
> In the case of a parsed url, you would need to enable this anyway as there is nothing else to tell otherwise.

you're right @stevenh. was making something more complex and buggy than it was required. fixed =)